File: Python/Beginners/idade_em_dias.py
# Caio Cesar 18/11/23
# BEE 1020

dias = int(input())

aux = dias % 365
anos = dias // 365
meses = aux // 30
dias = aux % 30

print(f"{anos} ano(s)")
print(f"{meses} mes(es)")
print(f"{dias} dia(s)")

File: C++/Beginner/idade_em_dias.cpp
// Caio Cesar 18/11/23
// BEE 1020

#include <cstdio>

int main() {
    int dias, aux, meses, anos;
    scanf("%d", &dias);

    aux = dias % 365;
    anos = dias / 365;
    meses = aux / 30;
    dias = aux % 30;

    printf("%d ano(s)\n", anos);
    printf("%d mes(es)\n", meses);
    printf("%d dia(s)\n", dias);

    return 0;

}